modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -137,6 +136,14 @@ const ClassicEditor = React.forwardRef(
 							}
 						}
 					}}
+					onInstanceReady={({editor}) => {
+						editor.setData(contents, {
+							callback: () => {
+								editor.resetUndo();
+							},
+							noSnapshot: true,
+						});
+					}}
```

Comment:
This partly reverts https://github.com/liferay-frontend/liferay-portal/pull/1133. In that PR, we fixed https://issues.liferay.com/browse/LPS-132871, with https://issues.liferay.com/browse/LPS-130112. Do we have regression in these cases?

Reply:
Tested https://issues.liferay.com/browse/LPS-132871, Thread body is visible, so no regression 💃:

https://user-images.githubusercontent.com/6642927/141310873-0cfd19ef-9ca9-45da-ba52-6a029aab9501.mov

Tested https://issues.liferay.com/browse/LPS-130112 in Web content and Message boards, `onChange` callback is not being triggered, so no regression 🎉 :

https://user-images.githubusercontent.com/6642927/141311242-1e24a21a-33ac-474d-9bec-914aef54f851.mov